client/src/templates/Challenges/components/__snapshots__/CompletionModalBody.test.js.snap

```diff
@@ -64,7 +64,7 @@ exports[`<CompletionModalBody /> matches snapshot 1`] = `
     <div
       class="completion-block-name"
     >
-      Basic HTML and HTML5
+      intro:responsive-web-design.blocks.basic-html-and-html5.title
```

Comment:
I don't think the key should ever appear in a component.  Shouldn't it fall back to the English?

Reply:
That's how it is configured, yes - but I'm not sure if the snapshots call the i18n function.

Reply:
Not sure if I implemented this part the best. The translations aren't actually loaded for [the tests](https://github.com/freeCodeCamp/freeCodeCamp/blob/main/client/i18n/configForTests.js). There's some [more info here](https://react.i18next.com/misc/testing). We could try and see if they can be loaded - I would have to play around with it. [Another relevant file](https://github.com/freeCodeCamp/freeCodeCamp/blob/main/client/src/__mocks__/react-i18nextMock.js)

Reply:
Okay, thanks for the context.  Don't worry about that for this PR, it's clearly out of scope.